pallets/committee/src/tests.rs

```diff
@@ -386,6 +387,31 @@ fn cannot_execute_proposal_twice() {
 	});
 }
 
+#[test]
+fn new_member_cannot_vote() {
+	new_test_ext(0..4).execute_with(|| {
+		let proposal = submit_proposal(123);
+		let proposal_hash = proposal.hash();
+		assert_ok!(Committee::add_constituent(Origin::root(), CONSTITUENT));
+
+		run_to_block(START_OF_V1 - 1);
+		vote_with_each(0..4, proposal.hash(), VoteKind::Aye);
+		assert_noop!(
+			Committee::vote(Origin::signed(CONSTITUENT), proposal_hash, VoteKind::Nay),
+			pallet::Error::<Test>::NotEligibileToVoteYet
+		);
+
+		run_to_block(START_OF_V1);
+		assert_noop!(
+			Committee::vote(Origin::signed(CONSTITUENT), proposal_hash, VoteKind::Nay),
```

Comment:
`vote` call doesn't check proposal status now, so we can only test the VotingPeriod

mb check proposals' status is better but `Proposal` and `VoteAggregate` just using different storages now

Reply:
> > vote call doesn't check proposal status now,
> 
> we do check it is in voting period?

Yep we only check if it is in voting period but no checks for if it is closed or timeout, for example, people try to vote and accept `NotInVotingPeriod` but they will never now if the proposal has been closed or timeout in this response